Assets/_BuildingBlocks/Credentials/Scripts/StoredAuthorization/QueryStringAuthorization.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using Netherlands3D.Web;
+
+namespace Netherlands3D.Credentials.StoredAuthorization
+{
+    [Serializable]
+    public class QueryStringAuthorization : StoredAuthorization
+    {
+        public string key = "";
```

Comment:
Private plz

Reply:
Na het lezen van de rest van de code zie ik dat dit nodig is voor de 3D Tiles handler, niet sjiek maar wel noodzakelijk voor nu; maar dit een readonly property maken helpt wel